Pirate-Wires/pirate-wires
# Refresh Router on Sign In/Out

### Notion Ticket
- https://www.notion.so/Session-issue-on-logout-the-navbar-still-says-account-and-users-can-t-comment-after-signing-in-4d0a5ba616704b22bc83d529f2b1b8b1?pvs=4

### QA Instructions
1. Run the app as a signed out user
2. In the top nav menu, click "Sign In" 
3. Complete the sign in process
4. When you arrive in the account UI (`/my-details` path) click `Pirate Wires` in the top nav menu, which will bring you back to the root path
5. Select any article
6. Leave a comment on the article
7. In the top nav menu, click "Account"
8. Click "Sign Out"
9. Verify that you are navigated back to the root path, and that the top nav menu reads `Account`
10. Navigate to an individual article
11. Verify that you are not able to leave a comment

https://github.com/Pirate-Wires/pirate-wires/assets/2637355/6e147ea2-63f2-4f9e-86d3-a87eae46a10c
